docs/content/querying/sql.md

```diff
@@ -77,6 +77,20 @@ If `druid.sql.planner.useFallback` is enabled, full SQL is possible on metadata
 recommended in production since it can generate unscalable query plans. The JDBC driver allows accessing
 table and column metadata through `connection.getMetaData()` even if useFallback is off.
 
+### Approximate queries
+
+The following SQL queries and features may be executed using approximate algorithms:
+
+- `COUNT(DISTINCT col)` aggregations use [HyperLogLog](http://algo.inria.fr/flajolet/Publications/FlFuGaMe07.pdf), a
+fast approximate distinct counting algorithm. If you need exact distinct counts, you can instead use
+`SELECT COUNT(*) FROM (SELECT DISTINCT col FROM druid.foo)`, which will use a slower and more resource intensive exact
```

Comment:
I also prefer this approach. Different behavior depending on query structure can make users confused.